met-api/src/met_api/services/submission_service.py

```diff
@@ -232,7 +232,7 @@ def get_paginated(cls, survey_id, pagination_options: PaginationOptions, search_
             search_text,
         )
         return {
-            'items': SubmissionSchema(many=True).dump(items),
+            'items': SubmissionSchema(many=True, exclude=['submission_json']).dump(items),
```

Comment:
isn't this what we use for public user editing the submission?

Reply:
this gets a page of submissions by survey id. It's used for comment listing, in both admin or public case submission json shouldn't be there.

This one below is used for public user editing the submission
![image](https://user-images.githubusercontent.com/91914654/230955094-6f827175-590a-41c5-a000-e3795a7491d8.png)